ofl/epilogue/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,7 @@
+<p>
+<em>Epilogue</em> is a sans serif variable font with a weight axis. 9 weights, upright and italic. It supports a wide range of languages in the latin script scope.
```

Comment:
@davelab6 do we support em tags?

Reply:
If you don't know, I can merge once it once I'm happy then check it on dev.

Reply:
Montserrat has an em tag